src/vcf/vcf_v43.ragel

```diff
@@ -884,15 +889,15 @@
                         CS record_alt 		%record_field_end $err(alt_error) 
                         CS record_qual		%record_field_end $err(qual_error) 
                         CS record_filter 	%record_field_end $err(filter_error) 
-                        CS record_info		%record_field_end $err(info_error) 
+                        CS record_info		%record_field_end $eof(body_section_no_nl_error) $err(info_error)
                         (CS record_format	%record_field_end $err(format_error)
-                        (CS record_sample	%record_field_end $err(sample_error))+ )?
+                        (CS record_sample	%record_field_end $eof(body_section_no_nl_error) $err(sample_error))+ )?
                       ) %record_end;
 
     fileformat_section = (fileformat NL) $err(fileformat_section_error);
     meta_section    = (meta_entry NL)* $err(meta_section_error);
     header_section  = (header NL) $err(header_section_error);
```

Comment:
Shouldn't there be a similar check to `$eof(body_section_no_nl_error)` for the NL after the header line? Maybe it could be converted to a more generic `$eof(no_newline_at_eof_error)`.